readme/BREAKING_CHANGES.markdown

```diff
@@ -381,4 +381,62 @@ away from soy:
 > hit the performance we wanted and more importantly, it always felt like we
 > were the only people using this technology.
 
+---------------------------------------
+
+### Server-side Closure Templates (Soy) Support has been removed
+- **Date:** 2020-Dec-14
+- **JIRA Ticket:** [LPS-122956](https://issues.liferay.com/browse/LPS-122956)
+
+#### What changed?
+
+The following modules and the classes they exported to allow Soy rendering
+server-side have been removed:
+- `portal-template-soy-api`
+- `portal-template-soy-impl`
+- `portal-template-soy-context-contributor`
+
+To simplify the migration, the following modules remain available in a deprecated
+deprecated fashion providing only client-side initialization of previous soy
```

Comment:
"deprecated deprecated"

also: should always capitalize the "s" in "soy"

Reply:
> also: should always capitalize the "s" in "soy"

You won't believe it, but I promise I double-checked all of them 😭